pyproject.toml

```diff
@@ -317,6 +317,9 @@ testing = ["dev", "providers.tests", "tests_common", "tests"]
 "providers/cncf/*/src/airflow/providers/cncf/__init__.py" = ["I002"]
 "providers/dbt/*/src/airflow/providers/dbt/__init__.py" = ["I002"]
 "providers/microsoft/*/src/airflow/providers/microsoft/__init__.py" = ["I002"]
+"providers/tests/__init__.py" = ["I002"]
+"providers/__init__.py" = ["I002"]
```

Comment:
```suggestion
```

Are these intentional? Probably not, but a miss during rebasing?

Reply:
Thanks! They weren't intentional and missed after the rebase. I rebased again and removed these additions.